torch/utils/collect_env.py

```diff
@@ -244,6 +253,17 @@ def get_os(run_lambda):
     # Unknown platform
     return platform
 
+def get_kernel_version():
+    import platform
+    if get_platform() == 'linux':
+        return '{} {}'.format(platform.release(), platform.version())
```

Comment:
would 
```
platform.platform()
```
gives more info?

Reply:
Good point, although it's not just a kernel version and can be a bit confusing, for example I have glibc-2.31 installed on my system, but `platform.platform()` reports glibc-2.2.5:
```
$ python -c "import platform;print(platform.platform());";/lib64/libc.so.6|head -n1
Linux-5.11.12-100.fc32.x86_64-x86_64-with-glibc2.2.5
GNU C Library (GNU libc) stable release version 2.31.
```

Reply:
yeah I was about to ask this question --> which libc do we actually care about?
for example in the conda environment there's a glibc that's different from non-conda env
```
Linux-5.2.9-229_fbk15_hardened_4185_g357f49b36602-x86_64-with-glibc2.28 -- from conda python3
Linux-5.2.9-229_fbk15_hardened_4185_g357f49b36602-x86_64-with-glibc2.10 -- from native python3
```